% LaTeX file for resume
% This file uses the resume document class (res.cls)
\documentclass[margin,line]{res} 
%\usepackage{helvetica} % uses helvetica postscript font (download helvetica.sty)
%\usepackage{newcent}   % uses new century schoolbook postscript font
\setlength{\textheight}{9.5in} % increase text height to fit on 1-page 
%% \setlength{\topmargin}{-0.25in}
\begin{document}

\name{\LARGE Kenneth Daily}
%% \address{\textbf{E-mail:} kenny.daily@nih.gov \textbf{Web:} www.kennydaily.net}
%% \address{10 Center Drive, Building 10 Room 12N202, Bethesda, MD 20892}
\address{\textbf{E-mail:} kenny@kennydaily.net \textbf{Web:} www.kennydaily.net}
\address{810 Taylor Ave N Apt 328, Seattle, WA 98109}

\begin{resume}

%% \section{\sc Objective}
%% Work in a collaborative environment building tools and performing computational\\
%% analyses of high-throughput genomic data to support biological research.

\section{\sc Current\\Position}
Senior Scientist, Data Science\\
Sage Bionetworks, Seattle, WA USA

\section{\sc Current\\Work}
I work on making biological science research reproducible, reusable, and sharable. This involves developing software for the Synapse platform (www.synapse.org) and working with research groups around the
world on best practices for collaborative research.

\section{\sc Education}
Ph.D., Computer Science, University of California, Irvine, 2011\\%% (Advisors: Prof. Pierre Baldi and Prof. Suzanne Sandmeyer)\\ 
M.S., Bioinformatics, Indiana University, 2006\\%% (Advisor: Prof. Predrag Radivojac)\\
B.S., Informatics, Indiana University, 2004%% (Advisor: Prof. Dennis Groth)

\section{\sc Research\\Interests}
Reproducible research, genomics, software and analysis pipeline development, data visualization and integration, data mining

\section{\sc Skills} 
\textbf{Languages:} R (Bioconductor), Python (Django, Ruffus, Snakemake), GNU Linux shell\\
\textbf{Applications:} Git, Galaxy, \LaTeX, Sun Grid Engine/PBS, Circos, MySQL\\
\textbf{Bioinformatics:} DNASeq, RNASeq, aCGH, microarray, TFBS prediction\\
\textbf{Methods:} Familiar with experimental biological techniques, frequentist and Bayesian statistics, machine learning

\section{\sc Current\\Projects}
%% \begin{itemize}
%% \itemsep -2pt % reduce space between items
Progenitor Cell Biology Consortium\\
NCI Up For A Challenge?\\
OHSU Microenvironment Pertubagens LINCS\\
Synapse client development
%% \end{itemize}

%% \section{\sc Past\\Projects}
%% \begin{itemize}
%% \itemsep -2pt % reduce space between items
%% Analysis of Transcriptional Regulation in Skin Cells Related to Wounding 2011 \\
%% Developed methods for analysis of genome-wide transcriptional regulation, 2011 \\
%% Analyzed high-throughput sequencing data of the yeast Ty3 transposon, 2009\\
%% Transcriptional Regulatory Control of Neural Stem Cell Development, 2011\\
%% Developed compression methods for high-throughput sequencing data, 2009 \\
%% Evaluated similarity metrics for chemical search, 2006\\
%% Predicted posttranslational protein modifications, 2004-2006
%% \end{itemize}

\section{\sc Publications}
N Salomonis, PJ Dexheimer, L Omberg, R Schroll, S Bush, J Huo, L Schriml, SH Sui, M Keddache, C Mayhew, D Witte, J Wells, \textbf{K Daily,} S Hubler, Y Wang, E Zambidis, A Margolin, W Hide, AK Hatzopoulos, P Malik, JA Cancelas, B Aronow, and C Lutzko. ``Integrated Genomic Analysis of Diverse Induced Pluripotent Stem Cell Lines Identifies Novel Molecular Determinants of Pluripotency.''. Submitted. \\
\textbf{K Daily}, T Alexander, I Brownell. ``Sequencing and characterization of Merkel cell polyomavirus integrations in Merkel cell carcinoma tumors.'' In preparation.\\
\textbf{K Daily}, A Coxon, DG Coit, KJ Busam, I Brownell. ``Genome-wide copy number analysis of Merkel cell carcinoma.'' In preparation.\\
\textbf{K Daily}, A Coxon, I Brownell. ``Identification of somatic changes in Merkel cell carcinoma by whole exome sequencing.'' In preparation.\\
\textbf{K Daily}, A Coxon, JS Williams, CR Lee, DG Coit, KJ Busam, I Brownell. ``Assessment of Cancer Cell Line Representativeness using Microarrays for Merkel Cell Carcinoma.'' J Invest Dermatol. Apr 2015. PMID: 25521454\\
AS Hopkin, W Gordon, RH Klein, F Espitia, \textbf{K Daily,} M Zeller, P Baldi, B Andersen. ``GRHL3/GET1 and Trithorax Group Members Collaborate to Activate the Epidermal Progenitor Differentiation Program.'' PLOS Genetics, July 2012. PMCID: PMC3400561\\
X Qi$^*$, \textbf{K Daily$^*$,} K Nguyen, H Wang, D Mayhew, P Rigor, S Forouzan, M Johnston, RD Mitra, P Baldi and S Sandmeyer. ``Retrotransposon profiling of RNA polymerase III initiation sites.'' Gen. Res., Apr 2012. PMCID: PMC331715 \let\thefootnote\relax\footnote{$^*$ Co-first authors} \\
\textbf{K Daily,} VR Patel, P Rigor, X Xie and P Baldi. ``MotifMap: integrative genome-wide maps of regulatory motif sites for model species.'' BMC Bioinfo., Dec 2011. PMCID: PMC3293935\\
\textbf{K Daily,} P Rigor, S Christley, X Xie, P Baldi. ``Data Structures and Compression Algorithms for High-Throughput Sequencing Technologies.'' BMC Bioinfo., Oct 2010. PMCID: PMC2964686\\
SJ Swamidass, CA Azencott, \textbf{K Daily,} P Baldi. ``A CROC stronger than ROC: measuring, visualizing and optimizing early retrieval.'' Bioinformatics, May 2010. PMCID: PMC2865862.\\
\textbf{K Daily,} P Radivojac, AK Dunker. ``Intrinsic disorder and protein modifications: building an SVM predictor for methylation.'' IEEE Symposium CIBCB 2005, San Diego, CA, Nov 2005.

\section{\sc Presentations}
``Oncogenomic analysis of Merkel cell carcinoma.'' Poster, ISMB. Boston, MA, 2014.\\
``The UISO cell line is not representative of Merkel cell carcinoma tumors.'' Poster, ISMB. Long Beach, CA, 2013.\\
``MotifMap: Genome-Wide Map of Regulatory Binding Sites.'' NLM Informatics Training Conference. Denver, CO, June 2010.\\
``Analysis of the Ty3 Retrotransposon in \textit{S. cerevisiae} Using Transposition Assays and HTS Technologies.'' NLM Informatics Training Conference. Portland, OR, June 2009.\\
``Data Structures and Compression Algorithms for High-Throughput Sequencing.'' Poster, ISMB. Stockholm, Sweden, June 2009.

\section{\sc Awards}
Fellows Award for Research Excellence, Center for Cancer Research, 2014\\
20th Annual Short Course on Experimental Models of Human Cancer, August 2011\\
Cancer Research Training Award, National Cancer Institute, 2011 - present\\
Graduate Deanâ€™s Dissertation Fellowship, UC Irvine, 2011\\
Biomedical Informatics Training Grant, National Library of Medicine, 2006 - 2010\\
Informatics Barwise Scholar, Indiana University, 2004 - 2006

%% \section{Teaching}
%% Teaching Assistant, University of California, Irvine, 2010 - 2011 \\
%% Associate Instructor, Indiana University School of Informatics, 2004 - 2006 \\
%% Course Instructor, Indiana University School of Informatics, Summer 2005

\section{\sc References} 

\textbf{Dr. Larsson Omberg, Ph.D.}\\
Principal Scientist\\
Sage Bionetworks\\
1100 Fairview Ave N\\
Seattle, WA 98109\\
E-Mail: larsson.omberg@sagebase.org\\

\textbf{Dr. Isaac Brownell, M.D., Ph.D.}\\
Investigator, Dermatology Branch\\
Center for Cancer Research\\
National Cancer Institute\\
Building 10, Room 12N246\\
Bethesda, MD 20892-1908\\
E-Mail: isaac.brownell@nih.gov\\


\end{resume}
\end{document}
