\documentclass[margin,line]{res}



\oddsidemargin -0.5in
\evensidemargin -0.5in
%% \topmargin -0.25in
\textheight=10in
\textwidth=6.0in
\itemsep=0in
\parsep=0in

\newenvironment{list1}{
  \begin{list}{\ding{113}}{%
      \setlength{\itemsep}{0in}
      \setlength{\parsep}{0in} \setlength{\parskip}{0in}
      \setlength{\topsep}{0in} \setlength{\partopsep}{0in} 
      \setlength{\leftmargin}{0.17in}}}{\end{list}}
\newenvironment{list2}{
  \begin{list}{$\bullet$}{%
      \setlength{\itemsep}{0in}
      \setlength{\parsep}{0in} \setlength{\parskip}{0in}
      \setlength{\topsep}{0in} \setlength{\partopsep}{0in} 
      \setlength{\leftmargin}{0.2in}}}{\end{list}}


\begin{document}

\name{Kenneth Daily \vspace*{.1in}}

\begin{resume}
\section{\sc Contact Information}
\vspace{.05in}
\begin{tabular}{@{}p{3in}p{4in}}
 10 Center Drive & {\it Voice:}  (301) 451-9240 \\
 Building 10, 12N252 &  {\it E-mail:} dailykm@mail.nih.gov\\         
 Bethesda, MD 20892 &  {\it Web:} www.kennydaily.net\\       
 & \\     
\end{tabular}


\section{\sc Research Interests}
Computational biology, cancer, genomics, high throughput sequencing,\\
regulation of transcription, data mining, machine learning

\section{\sc Experience}
        {\bf Center for Cancer Research, NCI, NIH}, Bethesda, MD USA\\
        \vspace*{-.1in}
        \begin{list1}
        \item[] Postdoctoral Fellow, Dermatology Branch, July 2011 - Present
          \begin{list2}
            \vspace*{.025in}
          \item Primary Investigator: Dr. Isaac Brownell, M.D., Ph.D.
          \end{list2}
          %% \vspace*{.025in}
        \end{list1}

\section{\sc Education}
        {\bf University of California, Irvine}, Irvine, California USA\\
        %{\em Department of Statistics} 
        \vspace*{-.1in}
        \begin{list1}
        \item[] Ph.D., Computer Science, September 2011
          \begin{list2}
            \vspace*{.025in}
          \item Advisors: Prof. Pierre Baldi, Prof. Suzanne Sandmeyer
          \end{list2}
          %% \vspace*{.05in}
        \end{list1}
        {\bf Indiana University}, Bloomington, Indiana USA\\
        \vspace*{-.1in}
        \begin{list1}
        \item[] M.S., Bioinformatics, June 2006
          \begin{list2}
            \vspace*{.05in}
          \item Advisor: Prof. Predrag Radivojac
          \end{list2}
          \vspace*{.05in}
        \item[] B.A., Informatics, June 2004           
          \begin{list2}
            \vspace*{.05in}
          \item Advisor: Prof. Dennis Groth
          \end{list2}
        \end{list1}
                  
        \section{\sc Honors and Awards} 
        Fellows Award for Research Excellence (Travel award), 2014. \\
        Cancer Research Training Award, National Cancer Institute, 2011 - present.\\
        Graduate Deanâ€™s Dissertation Fellowship, UC Irvine, 2011\\
        Best Presentation, NLM Informatics Training Conference. Denver, CO, June 2010.\\
        Travel Award, ISCB ISMB, Stockholm, Sweden, June 2009.\\
        Biomedical Informatics Training Grant, National Library of Medicine, 2006 - 2010.\\
        Travel Award, IEEE CIS CIBCB, November 2005.\\ %% Computational Intelligence in Bioinformatics and Computational Biology
        Informatics Barwise Scholar, Indiana University, 2004 - 2006.

        \section{\sc Publications}
        Xiaojie Qi, \textbf{Kenneth Daily,} Kim Nguyen, Haoyi Wang, David Mayhew, Paul Rigor, Sholeh Forouzan, Mark Johnston, Robi David Mitra, Pierre Baldi and Suzanne Sandmeyer. ``Retrotransposon profiling of RNA polymerase III initiation sites.'' Genome Research, April 2012. PMCID: PMC3317150
        
        \textbf{Kenneth Daily,} Vishal R Patel, Paul Rigor, Xiaohui Xie and Pierre Baldi. "MotifMap: integrative genome-wide maps of regulatory motif sites for model species." BMC Bioinformatics December 2011.PMCID: PMC3293935

        \textbf{Kenneth Daily,} Paul Rigor, Scott Christley, Xiaohui Xie, Pierre Baldi. ``Data Structures and Compression Algorithms for High-Throughput Sequencing Technologies.'' BMC Bioinformatics, October 2010. PMCID: PMC2964686

        S. Joshua Swamidass, Chloe-Agathe Azencott, \textbf{Kenneth Daily,} Pierre Baldi. ``A CROC stronger than ROC: measuring, visualizing and optimizing early retrieval.'' Bioinformatics, May 2010. PMCID: PMC2865862.
        
        \textbf{Kenneth Daily,} Predrag Radivojac, A. Keith Dunker. ``Intrinsic disorder and protein modifications: building an SVM predictor for methylation.'' IEEE Symposium on Computational Intelligence in Bioinformatics and Computational Biology, CIBCB 2005, pp. 475-481, San Diego, California, U.S.A., November 2005.
        
        %% \section{\sc Papers in preparation}
        %% \textbf{Kenneth Daily,} Paul Rigor, Sholeh Forouzan, Yimeng Dou, Vishal Patel, Xiaohui Xie and Pierre Baldi. ``Genome-wide maps of candidate regulatory motif sites.'' In Preparation.
        
        %% Suzanne Sandmeyer, Kim Nguyen, \textbf{Kenneth Daily,} Xiaojie Qi, Pierre Baldi. ``Analysis of the Ty3 Retrotransposon in Saccharomyces cerevisiae Using Transposition Assays and High Throughput Sequencing Technologies.'' In Preparation.
        
        %% W. Jacob B. Biesinger, Jane Cho, \textbf{Kenneth Daily,} Paul Rigor, Douglas Bornemann, Rahul Warrior, Pierre Baldi, Kavita Arora, and Xiaohui Xie. ``A Map of Candidate Regulatory Motifs in Fly.'' In Preparation.
        
        \newpage

        \section{\sc Conference Presentations}
        ``The UISO cell line is not representative of Merkel cell carcinoma tumors.'' Poster, Great Lakes Bioinformatics Conference. Pittsburgh, PA, 2013.

        ``MotifMap: Genome-Wide Map of Regulatory Binding Sites.'' Poster, Society for Investigative Dermatology Annual Meeting. Raleigh, NC, 2012.

        ``MotifMap: Genome-Wide Map of Regulatory Binding Sites.'' Poster, HTS to P4 Medicine Conference. UC Irvine, Irvine, CA, February 2011.
        
        ``MotifMap: Genome-Wide Map of Regulatory Binding Sites.'' NLM Informatics Training Conference. Denver, CO, June 2010.

        ``Analysis of the Ty3 Retrotransposon in Saccharomyces cerevisiae Using Transposition Assays and High Throughput Sequencing Technologies.'' NLM Informatics Training Conference. Portland, OR, June 2009.

        ``Data Structures and Compression Algorithms for High-Throughput Sequencing.'' Poster, International Conference on Intelligent Systems for Molecular Biology. Stockholm, Sweden, June 2009.

        \section{\sc Teaching}
        Teaching Assistant, University of California, Irvine, 2010 - 2011 \\
        Associate Instructor, Indiana University School of Informatics, 2004 - 2006 \\
        Course Instructor, Indiana University School of Informatics, Summer 2005
                
        \section{\sc Skills} 
        \begin{list2}
        \item Languages: Python (TurboGears, Django), R (Bioconductor), Gnu Linux/Unix shell, Perl (with BioPerl), Matlab, PHP, Java, HTML (CSS), C/C++, SQL, XML.
        \item Applications: Git, \LaTeX, MySQL, Sun Grid Engine/PBS
        \item Operating Systems:  Linux/Unix, Windows.
        \item Bench work: Basic molecular biology methods for bacteria, yeast, fly, and mouse ([rt]PCR, gel electrophoresis, primer design, transformation, DNA extraction and sequencing,\\ Northern/Western/Southern blots, mouse handling, genotyping)
        \end{list2}

        \section{\sc References} 
        \textbf{Professor Pierre Baldi}
        \begin{list2}
          \item[] School of Information and Computer Sciences
          \item[] 4038 Bren Hall
          \item[] University of California, Irvine
          \item[] Irvine, CA 92697-3435
          \item[] E-mail: pfbaldi@ics.uci.edu
          \item[] {\it WWW:} http://www.ics.uci.edu/\verb+~+pfbaldi
        \end{list2}

        \textbf{Professor Suzanne Sandmeyer}
        \begin{list2}
          \item[] Department of Biological Chemistry
          \item[] University of California, Irvine
          \item[] D240, D288 MED SCI I
          \item[] Mail Code: 1700
          \item[] Irvine, CA 92697
          \item[] E-mail: sbsandme@uci.edu
          \item[] {\it WWW:} http://webfiles.uci.edu/sbsandme/lab/
        \end{list2}

\end{resume}
\end{document}




